### PR TITLE
PlotNAB class improvements

### PR DESCRIPTION
**Pull request by BoltzmannBrain**

### CODE REVIEW

**Comment by BoltzmannBrain** [comment]
fixes #155

**Comment by BoltzmannBrain** [comment]
@scottpurdy addresses your previous review

File: scripts/plot.py [hunk 1]
```diff
@@ -27,7 +27,15 @@
 import pandas as pd
 import plotly.plotly as py
 
-from plotly.graph_objs import *
+from plotly.graph_objs import (Bar,
```
Comment (scottpurdy) [inline scripts/plot.py]: yikes, I'd break the first `Bar` onto a new line and then list all of the imported items on the same line, breaking only if you hit the end of the line.

alternatively, you could do `from plotly import graph_objs` and then later do `graph_objs.Bar`, etc.

File: scripts/plot.py [hunk 2]
```diff
@@ -392,33 +399,33 @@ def _addDetections(self, name, symbol, FP, TP):
   # Examples:
   
   dataFiles = [
```
Comment (scottpurdy) [inline scripts/plot.py]: You could make these tuples instead of lists so they are explicitly immutable.

**Comment by scottpurdy** [comment]
:+1:

**Pull request merged**